packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -1164,12 +1164,15 @@ class DisplayOptionsStatusExample extends React.Component<{}> {
 function DisplayOptionStatusExample({optionName, optionChecker, notification}) {
   const [statusEnabled, setStatusEnabled] = React.useState(false);
   React.useEffect(() => {
-    AccessibilityInfo.addEventListener(notification, setStatusEnabled);
+    let listener = AccessibilityInfo.addEventListener(
```

Comment:
It's better to you `const` instead of `let` here

Reply:
I could see some issues in linting while using const in different scopes. I will change it to const.

Reply:
Fixed [ad7a226](https://github.com/facebook/react-native/pull/33580/commits/ad7a226bdacecea557001876dc7dcbcfb82075a1).